### PR TITLE
Properly style night mode dialogs

### PR DESCRIPTION
**Pull request by ashutoshbsathe**
Holo theme creates issue on newer android versions creating extra space above title in the alert dialogs.

Fixes : #517 

Changes: New styles were added for night mode 

Screenshots/GIF for the change: Final result is shown here
screenshot_1519709371
screenshot_1519710526
@brijeshshah13 @mhutti1 do review and let me know if any more changes are required

### CODE REVIEW

**Comment by ashutoshbsathe** [comment]
@brijeshshah13 Done :D 
screenshot_1519727035

**Comment by MayankChuriwal** [comment]
I think the white color was looking better as the blue color is contrasting with the back background and is not quite visible. It would be better if the changes are reverted back to white color. @ashutoshbsathe @brijeshshah13

**Comment by brijeshshah13** [comment]
@MayankChuriwal Actually those are the standards for the dark theme. However, the blue color used by Kiwix is a little bit dark as opposed to what it should be. Here, a sample for the same:

screenshot_20180227-194332

**Comment by brijeshshah13** [comment]
@mhutti1 LGTM. Tested on API 25 device and API 16 emulator.

**Comment by MayankChuriwal** [comment]
Ya that looks perfect, I meant the same that the blue right which is used now is not proper so I suggested to use white instead of this color. @brijeshshah13

**Comment by brijeshshah13** [comment]
@MayankChuriwal I will raise a new issue to change the color where ever necessary.

**Comment by MayankChuriwal** [comment]
@brijeshshah13 I might be wrong but I think color of dialog box does not come under material design guidelines by Google.
Here a link to the material design guidelines for Dialog: https://material.io/guidelines/components/dialogs.html#dialogs-specs

Moreover many apps of Google itself uses different colors for dialog buttons:
dialog_alert_simple
banner

Do let me know if I am wrong.

**Comment by brijeshshah13** [comment]
@MayankChuriwal As i stated, those are the **standards** for the dark theme and not the Material Design Guidelines provided by Google. The reason why almost all the apps use a different color for action buttons (in this case blue) for the dark theme is because it separately signifies the options available to the users. However, it is not compulsory to use the same color. And the screenshots you provided are of the regular mode and not of the night mode.

**Comment by MayankChuriwal** [comment]
@brijeshshah13 I got your points and thanks for the information.
I have made changes to the color codes as per the standards and going through the standards I came up with two solutions.

This is what you were talking about: 
blue

And I think this is the standard color for alert dialog button
whatsapp image 2018-02-27 at 10 19 46 pm

**Comment by brijeshshah13** [comment]
@MayankChuriwal I think we should go with the first one as we are using blue for the titles in the settings as well. I have created a new issue #532 to deal with this. Once @mhutti1 verifies, you can go ahead and fix it.

File: app/src/main/res/drawable/ic_warning_black.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"
```
Comment (brijeshshah13) [inline app/src/main/res/drawable/ic_warning_black.xml]: 2 space indentation

File: app/src/main/res/drawable/ic_warning_white.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FFFFFFFF"
```
Comment (brijeshshah13) [inline app/src/main/res/drawable/ic_warning_white.xml]: 2 space indentation

File: app/src/main/res/drawable/ic_warning_black.xml (hunk 1 above)
Comment (ashutoshbsathe) [reply app/src/main/res/drawable/ic_warning_black.xml]: Oh yes ! I completely forgot to indent these 
New commit soon !

File: app/src/main/res/drawable/ic_warning_black.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24.0"
+    android:viewportHeight="24.0">
+  <path
+      android:fillColor="#FF000000"
```
Comment (brijeshshah13) [inline app/src/main/res/drawable/ic_warning_black.xml]: I'm still counting 4 spaces indentation.

File: app/src/main/res/drawable/ic_warning_white.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+   android:width="24dp"
+   android:height="24dp"
+   android:viewportWidth="24.0"
+   android:viewportHeight="24.0">
+  <path
+      android:fillColor="#FFFFFFFF"
```
Comment (brijeshshah13) [inline app/src/main/res/drawable/ic_warning_white.xml]: 4 spaces indentation

File: app/src/main/res/drawable/ic_warning_black.xml (hunk 3 above)
Comment (ashutoshbsathe) [reply app/src/main/res/drawable/ic_warning_black.xml]: Oops, VI somehow expanded tabs to 4 spaces instead of 2 spaces before push :/
Fixed now.

Comment (brijeshshah13) [reply app/src/main/res/drawable/ic_warning_black.xml]: @ashutoshbsathe No problem. @mhutti1 will review it soon.

File: app/src/main/java/org/kiwix/kiwixmobile/settings/KiwixSettingsActivity.java [hunk 5]
```diff
@@ -275,6 +275,11 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
     }
 
     private void clearAllHistoryDialog() {
+      int warningResId;
+      if(nightMode(PreferenceManager.getDefaultSharedPreferences(getActivity())))
+        warningResId = R.drawable.ic_warning_white;
```
Comment (mhutti1) [inline app/src/main/java/org/kiwix/kiwixmobile/settings/KiwixSettingsActivity.java]: Styling. Spacing is off. Also curtly brackets are very much preferred especially with an else stament.

Comment (ashutoshbsathe) [reply app/src/main/java/org/kiwix/kiwixmobile/settings/KiwixSettingsActivity.java]: I'm sorry but I didn't quite understand what you meant by "Styling. Spacing is off"
Do you need space between <code>if</code> and <code>(</code> like we have in <code>onSharedPreferenceChanged</code> method ?

Comment (RajaVamsi11) [reply app/src/main/java/org/kiwix/kiwixmobile/settings/KiwixSettingsActivity.java]: Use  curly brackets for if-else statement.

Comment (mhutti1) [reply app/src/main/java/org/kiwix/kiwixmobile/settings/KiwixSettingsActivity.java]: And the space as well. Consistent style is very important.

**Comment by ashutoshbsathe** [comment]
@mhutti1 @brijeshshah13 
Everything is done !
Do review and merge

**Pull request merged**